runhouse/servers/cluster_servlet.py

```diff
@@ -321,7 +328,7 @@ async def aperiodic_cluster_checks(self):
                     f"If a value is not provided, interval size will be set to {round(DEFAULT_STATUS_CHECK_INTERVAL / 60, 2)} minutes."
                 )
                 await asyncio.sleep(INCREASED_STATUS_CHECK_INTERVAL)
-            else:
+            finally:
```

Comment:
maybe i misunderstood, but why was this previously an `else`?

Reply:
no particular reason, finally indeed makes more sense :)